RevenueCatUI/CustomerCenter/ViewModels/BaseManageSubscriptionViewModel.swift

```diff
@@ -225,7 +225,7 @@ extension BaseManageSubscriptionViewModel {
     var changePlanProductIDs: [String] {
         purchaseInformation?
             .changePlan
-            .map { $0.products.map(\.productId) } ?? []
+            .map { $0.products.filter { $0.selected }.map(\.productId) } ?? []
```

Comment:
Even though khepri is sending only selected products now, I think this won't hurt